components/registry-facade/pkg/registry/manifest.go

```diff
@@ -208,6 +210,14 @@ func (mh *manifestHandler) getManifest(w http.ResponseWriter, r *http.Request) {
 			manifest.Config.URLs = nil
 			manifest.Config.Size = int64(len(rawCfg))
 
+			// We might have additional modifications, e.g. adding IPFS URLs to the layers
+			if mh.ManifestModifier != nil {
+				err = mh.ManifestModifier(manifest)
+				if err != nil {
+					log.WithError(err).WithFields(logFields).Warn("cannot modify manifest")
```

Comment:
worth printing the error in the log statement?

Reply:
We are printing the error at the moment - should we not do that? If so, why?